scripts/release/create-github-draft-release.sh

```diff
@@ -27,6 +27,7 @@ main() {
 
     MAPBOX_COMMON_VERSION=$(jq -r .MapboxCommon "$VERSION_JSON_PATH")
     MAPBOX_COREMAPS_VERSION=$(jq -r .MapboxCoreMaps "$VERSION_JSON_PATH")
+    XCODE_MIN_VERSION=$(python3 -c "import yaml,sys;print(yaml.safe_load(sys.stdin)['executors']['xcode-sdk-min']['macos']['xcode'])" < "$SCRIPT_DIR/../../.circleci/config.yml")
```

Comment:
Could you add a comment here describing what's happening and also a comment into the circleci config so that folks will know there's an external script that depends on the structure.